CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### Bug fixes
+
+* [#9208](https://github.com/rubocop-hq/rubocop/issues/9208): Use Array#bsearch instead of Array#include? to detect hidden files. ([@dark-panda][])
```

Comment:
I assume the build is failing, because you forgot to add a link to your handle to the bottom of the file. Or you can just generate the changelog entry as @dvandersluis suggested. This is the way. 😆

Reply:
Ah, yeah, that's likely it. Will fix it up now.